### ISSUE TITLE
Download fails

### ISSUE THREAD

**Issue by kvellano** [cwmcbrewster/Adobe#1]
it looks like Adobe has changed the download URL? Browsing to:

https://armmf.adobe.com/arm-manifests/mac/AcrobatDC/acrobat/current_version.txt

The version doesn't seem to work in the previously used URL:

http://ardownload2.adobe.com/pub/adobe/reader/mac/AcrobatDC/<CurrentVersionShort>

**Comment by cwmcbrewster** [comment]
It's working fine here. Note how the script uses currentVersion to create currentVersionShort, which is then used in the download URL.

**Comment by kvellano** [comment]
Yes, sorry - disregard!

**Comment by kvellano** [comment]
This is sort of bizarre. If I call the shell script from a host - no issues. If I deploy with Jamf Pro, I get:

```
Script exit code: 1
Script result: Current version: 2100120149
Temp dir set to /var/folders/zz/zyxvpxvq6csfxvn_n0000000000000/T/tmp.Q0yv3saS
Adobe Acrobat Reader DC.app not installed
AdobeReader not currently running
hdiutil: attach failed - image not recognized
installer: Error - the package path specified was invalid: '/Volumes/AcroRdrDC_2100120149_MUI/AcroRdrDC_2100120149_MUI.pkg'.
hdiutil: detach failed - No such file or directory
Adobe Acrobat Reader DC.app not installed
```

**Comment by cwmcbrewster** [comment]
Grab a current copy of the script. Adobe released the 2100120149 version as an update only without a full installer which broke things.

**Comment by kvellano** [comment]
Yes, it appears so - thank you!
